admin-ui/plugins/fido/components/Fido.tsx

```diff
@@ -35,25 +35,24 @@ const Fido = () => {
   )
 
   const handleStaticConfigurationSubmit = useCallback(
-    (data) => {
+    (data: FormData) => {
       const apiPayload = createFidoConfigPayload({
         fidoConfiguration,
         data,
         type: fidoConstants.STATIC,
       })
-
       dispatch(putFidoConfiguration(apiPayload))
     },
     [dispatch, fidoConfiguration],
   )
 
-  const tabNames = [
+  const tabNames: TabName[] = [
     { name: t('menus.static_configuration'), path: '/fido/fidomanagement/static-configuration' },
     { name: t('menus.dynamic_configuration'), path: '/fido/fidomanagement/dynamic-configuration' },
   ]
```

Comment:
Memoize tabNames to avoid re-creating the array on each render.

Minor optimization; keeps GluuTabs from unnecessary re-renders when t doesn’t change.

```diff
-  const tabNames: TabName[] = [
-    { name: t('menus.static_configuration'), path: '/fido/fidomanagement/static-configuration' },
-    { name: t('menus.dynamic_configuration'), path: '/fido/fidomanagement/dynamic-configuration' },
-  ]
+  const tabNames = React.useMemo<TabName[]>(
+    () => [
+      { name: t('menus.static_configuration'), path: '/fido/fidomanagement/static-configuration' },
+      { name: t('menus.dynamic_configuration'), path: '/fido/fidomanagement/dynamic-configuration' },
+    ],
+    [t],
+  )
```
Note: Add useMemo to imports if not already present.